languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,14 @@
+In C# and in many other languages, the `null` literal denotes the abscence of a value.
```

Comment:
Could you also introduce the `NullReferenceException` somewhere in this file, as well as the various null-related operators?